redbeams-api/src/test/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/responses/DatabaseServerV4ResponsesTest.java

```diff
@@ -0,0 +1,28 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+
+import org.junit.Test;
+
+public class DatabaseServerV4ResponsesTest {
+
+    private DatabaseServerV4Responses responses;
+
+    @Test
+    public void testEmpty() {
+        responses = new DatabaseServerV4Responses();
+        assertEquals(0, responses.getResponses().size());
+    }
+
+    @Test
+    public void testNonEmpty() {
+        DatabaseServerV4Response response = new DatabaseServerV4Response();
+        response.setId(1L);
+
+        responses = new DatabaseServerV4Responses(Collections.singleton(response));
+        assertEquals(1, responses.getResponses().size());
+        assertEquals(1L, responses.getResponses().iterator().next().getId().longValue());
```

Comment:
Just a tip: with hamcrest there are collection matchers. It includes matchers like
- hasElement
- hasField
https://stackoverflow.com/questions/12166415/how-do-i-assert-an-iterable-contains-elements-with-a-certain-property

Reply:
Thanks! I'm sure someone (probably me) will revisit these unit tests at a later time, so I'll remember to improve the assertions at that time.